node/scripts/rewrite_package_json.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python
+
+#
+# Copyright 2021 Signal Messenger, LLC.
+# SPDX-License-Identifier: AGPL-3.0-only
+#
+
+import sys
+import json
+
+def remove_subdir(l):
+    l = l.replace('node/', '')
+
+    if l == "tsc -b node":
+        l = "tsc"
```

Comment:
There won't be any TypeScript to build, so it shouldn't matter what the shortcut is. Or is it getting run anyway?

Why bother flattening the paths at all? It's going to be embedded inside the app anyway.

Reply:
TBH logic here is 

I have no #### clue what yarn may or may not do with a `package.json` it pulls from a dependency
With `node/` prefix it obviously doesn't work with respect to the layout in libsignal-client-node
Maybe yarn doesn't look at any of this in which case rewriting it is harmless

Reply:
Right, I'm saying that the layout in libsignal-client-node may as well match the layout in libsignal-client. (The "main" entry in the package.json should make that okay; "dist" isn't magic either.)